src/Build/Logging/BinaryLogger/BinaryLogger.cs

```diff
@@ -28,10 +28,12 @@ public sealed class BinaryLogger : ILogger
         // version 5:
         //   - new EvaluationFinished.ProfilerResult
         // version 6:
-        //   -  Ids and parent ids for the evaluation locations
+        //   - Ids and parent ids for the evaluation locations
         // version 7:
-        //   -  Include ProjectStartedEventArgs.GlobalProperties
-        internal const int FileFormatVersion = 7;
+        //   - Include ProjectStartedEventArgs.GlobalProperties
+        // version 8:
+        //   - new record kinds: EnvironmentVariableRead, PropertyReassignment, UninitializedPropertyRead
+        internal const int FileFormatVersion = 8;
 
```

Comment:
Later on in this file (`Initialize` method) we should probably set MsBuildLogPropertyTracking=1 so that the binary logger gets the data.

Reply:
Added this to latest commit.